common/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/common/model/PrisonerTest.kt

```diff
@@ -0,0 +1,77 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.common.model
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+
+class PrisonerTest {
+  @Test
+  fun `all Prisoner fields are nullable`() {
+    assertDoesNotThrow {
+      jacksonObjectMapper().readValue("{}", Prisoner::class.java)
```

Comment:
bit naff to instantiate this each time and might be different from the one created by spring.  Would be better to use `@JsonTest` and then you'll get the correct one injected.  See https://docs.spring.io/spring-boot/reference/testing/spring-boot-applications.html#testing.spring-boot-applications.json-tests.  Even if you don't use the `JacksonTester` it will still give you an object mapper.